docs/scripts_md/BackPopulateSNRAndAcquisitionOrder.md

```diff
@@ -0,0 +1,45 @@
+# NAME
+
+BackPopulateSNRAndAcquisitionOrder.pl -- a script that back populates the
+AcqOrderPerModality column of the files table, and the signal-to-noise (SNR)
+values in the parameter\_file table for inserted MINC files. The SNR is computed
+using algorithms built-in the MINC tools.
```

Comment:
maybe rephrase "algorithms built-in the MINC tools" to "MINC tools built-in algorithms"?

Reply:
done